plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/AbstractPinotIntegrationConnectorSmokeTest.java

```diff
@@ -141,6 +143,30 @@ protected QueryRunner createQueryRunner()
         TestingPinotCluster pinot = closeAfterClass(new TestingPinotCluster(kafka.getNetwork(), isSecured(), getPinotImageName()));
         pinot.start();
 
+        createAndPopulateAllTypesTpoic(kafka, pinot);
+        createAndPopulateMixedCaseTableAndTopic(kafka, pinot);
+        createAndPopulateMixedCaseDistinctTableAndTopic(kafka, pinot);
+        createAndPopulateTooManyRowsTable(kafka, pinot);
+        createAndPopulateTooManyBrokerRowsTableAndTopic(kafka, pinot);
+        createTheDuplicateTablesAndTopics(kafka, pinot);
+        createAndPopulateDateTimeFieldsTableAndTopic(kafka, pinot);
+        createAndPopulateJsonTable(kafka, pinot);
+        createAndPopulateMixedCaseHybridTablesAndTopic(kafka, pinot);
+        createAndPopulateTableHavingReservedKeywordColumnNames(kafka, pinot);
+        createAndPopulateHavingQuotesInColumnNames(kafka, pinot);
+        createAndPopulateHavingMultipleColumnsWithDuplicateValues(kafka, pinot);
+        createAndPopulateNationAndRegionData(kafka, pinot);
```

Comment:
Could you add a preparatory commit for introducing these helper methods?

Reply:
Thanks for the review. The initial commit is having this very change, preparatory commit could be a bit tedious task by segregating the commits and trying to recreate only this section. Is it very much important to do so? If yes, I can then maybe redo this PR with a separate branch (which is easier). Let me know your thoughts.

Reply:
The commit granularity is important. Please see https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#git-merge-strategy

I hope you will handle in this PR instead of a new PR.

Reply:
Have addressed the first commit to only have refactored methods.